packages/editor-ui/src/components/RunData.vue

```diff
@@ -995,23 +995,16 @@ export default defineComponent({
 
 			this.clearAllStickyNotifications();
 
-			if (!this.isValidPinDataSize(value)) {
-				this.onDataPinningError({ errorType: 'data-too-large', source: 'save-edit' });
-				return;
-			}
-
-			if (!this.isValidPinDataJSON(value)) {
-				this.onDataPinningError({ errorType: 'invalid-json', source: 'save-edit' });
+			const pinDataSuccess = this.setPinData(
+				this.node,
+				clearJsonKey(value) as INodeExecutionData[],
```

Comment:
I'm not sure about this one. We're not using the `clearJsonKey` in every place that we're using `setPinData`. Do you think it could have any side effects?